items/weapons.md

```diff
@@ -0,0 +1,51 @@
+# Weapons
```

Comment:
I'd suggest a structural change.
Have:
- systems/items.md - for defining basic items
- systems/weapons.md - for defining basic weapons
And additional tables of items would be outside of the system - ie, setting dependent. These are outside the scope of the system itself.

I am open for discussion on this.

Reply:
Hmm, yea. I was just trying to not have all of everything in one file. I was thinking there would be a file for each item type which would make things messy, but we could do it like this, and maybe put general items in `items.md`

Reply:
Yes. If we have additional lists of items, such as lists of potions, ect, then we can move it to items/potions.md.
But imo the base items.md should be in systems/.